tests/src/test/java/alluxio/client/RemoteBlockInStreamIntegrationTest.java

```diff
@@ -82,6 +82,13 @@
     return list;
   }
 
+  /**
+   * Constructor for {@link RemoteBlockInStreamIntegrationTest}.
+   *
+   * @param dataServer the address of the worker's data server
+   * @param transferType the file transfer type used by the worker
+   * @param reader never used
```

Comment:
since this is never used, could you remove it from the signature and delete the third arguments in the `data()` method?